### ISSUE TITLE
Double extending seems to not be working

### ISSUE THREAD

**Issue by nombrekeff** [RobotUnion/rallf-js-sdk#72]
If you extend from another class that extends from `rallf.Task` warmup breaks.

```js
class Test extends rallf.Task {
  warmup(){
    // Works
  }
}

class TestExtends extends Test {
  warmup(){
    // Does not work, it does not call this.warmup neither super.warmup
  }
}
```

**Comment by nombrekeff** [comment]
Add more tests for this

**Comment by nombrekeff** [comment]
Did not work is runned with global command

**Comment by nombrekeff** [comment]
Should be fixed as of v0.11.0
